ttnn/cpp/ttnn/operations/examples/example/device/example_device_operation.hpp

```diff
@@ -135,5 +135,13 @@ struct ExampleDeviceOperation {
         tensor_return_value_t&);
     */
 };
+struct ExampleOperation {
+    // Map API arguments to the device operation
+    using device_operation_t = ExampleDeviceOperation;
+    using tensor_args_t = device_operation_t::tensor_args_t;
+    using operation_attributes_t = device_operation_t::operation_attributes_t;
+
+    static std::tuple<operation_attributes_t, tensor_args_t> map_args_to_device_operation(const Tensor& input_tensor);
```

Comment:
this maps to operator()?

Reply:
when the op gets registered `operator()` of the registered object maps to device operation via `map_args_to_device_operation`